services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_base_scheduler.py

```diff
@@ -929,22 +929,25 @@ async def _schedule_tasks_to_start(  # noqa: C901
                 comp_tasks[NodeIDStr(f"{task}")].state = RunningState.FAILED
             raise
         except TaskSchedulingError as exc:
+            err_context = exc.error_context()
             _logger.exception(
                 "Project '%s''s task '%s' could not be scheduled",
-                exc.project_id,
-                exc.node_id,
+                err_context["project_id"],
+                err_context["node_id"],
             )
             await CompTasksRepository.instance(
                 self.db_engine
             ).update_project_tasks_state(
                 project_id,
-                [exc.node_id],
+                [err_context["node_id"]],
                 RunningState.FAILED,
-                exc.get_errors(),
+                None,  # exc.get_errors(), # @pcrespov I need your help here!
```

Comment:
Let's talk about this offline. In short, current `OsparcErrorMixin` does not contain this member. but there is a way around

Reply:
fixed after discussion